src/js/components/Grommet/Grommet.js

```diff
@@ -39,6 +39,9 @@ class Grommet extends Component {
       } else if (nextTheme.dark === undefined) {
         nextTheme.dark = (background && colorIsDark(background)) || false;
       }
+      if (direction) {
+        nextTheme.direction = direction;
```

Comment:
should we replace `direction` with `dir`?

Reply:
yes